### DISCUSSION TITLE
Linking recipes within a recipe

### DISCUSSION THREAD

**Discussion by inter5tella** [mealie-recipes/mealie#4627] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I want to link a already existing recipe in the ingredients notes or the recipe notes.

E.g. I want to create a sauce recipe and another recipe is already covering the base ingredients/instructions, I want to show a link that will open this recipe.

Instead of re-adding the same ingredients and steps of an already existing recipe, I want to be able to add a link into the ingredients notes or the recipe steps component to quickly open an already existing recipe page.

### Please provide a concise description of the feature that would resolve your issue.

Adding a function to use a specific character (e.g. '@' or '#') allows me to see a picklist of existing recipes sorted by name, that will add a hyperlink to '<local address>/g/home/r/xxx' as destination. xxx represents the actual selected db recipe.

### Please consider and list out some caveats or tradeoffs made in your design decision

It could happen that the list of recipes to select is too long and might break the UI. Maybe it should be limited to only 20 items and will be reloaded for the next 20 items after reaching the bottom of the picklist.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by FuzzyWereBeast** [comment]
I use markdown for the ingredient.  In your example it would be `[sauce](.g/home/r/sauce)`.  This allows you to just click on the ingredient to go to the recipe.

**Comment by Kuchenpirat** [comment]
Besides linking the recipe via the method described above this is a Duplicate of #1761
